indigo_api/models.py

```diff
@@ -1262,6 +1263,7 @@ def submit(self, user):
             self.assign_to(user, user)
         self.last_assigned_to = self.assigned_to
         self.assigned_to = None
+        self.save()
```

Comment:
should https://github.com/laws-africa/indigo/blob/master/indigo_app/views/tasks.py#L245 move here
and https://github.com/laws-africa/indigo/blob/master/indigo_app/views/tasks.py#L266 come out?

Reply:
ditto for updated_by_user below × 4

Reply:
Hmm. I think L266 (the `.save()`) can be removed.

I'm happy if the `updated_by_user` stays in the view. This method doesn't know which user is doing the updating, necessarily.